tasks/taxon_id/task_gambit.wdl

```diff
@@ -181,3 +181,167 @@ task gambit {
     preemptible:  0
   }
 }
+
+task gambit_euk {
+  input {
+    File assembly
+    String samplename
+    String docker = "quay.io/staphb/gambit:0.5.0"
+    File gambit_db_genomes = "gs://theiagen-public-files/terra/candida_auris_refs/221006-theiagen-fungal-v0.1.db"
+    File gambit_db_signatures = "gs://theiagen-public-files/terra/candida_auris_refs/221006-theiagen-fungal-v0.1.h5" 
+  }
+  # If "File" type is used Cromwell attempts to localize it, which fails because it doesn't exist yet.
```

Comment:
not 100% what this is referencing -- is this an issue still or a residual comment?

Reply:
Oh! Is it talking about the variables below?

Reply:
I get it, nevermind